modules/pam_mkhomedir/pam_mkhomedir.c

```diff
@@ -170,6 +174,9 @@ create_homedir (pam_handle_t *pamh, options_t *opt,
 	args[2] = opt->umask ? opt->umask : UMASK_DEFAULT;
 	args[3] = opt->skeldir;
 	args[4] = login_homemode;
+   if (opt->use_copy_mode == 1) {
+      args[5] = "copymode";
+   }
```

Comment:
Indentation seems off …

Reply:
Thanks for the review. 
Indentation turned to on with the last change.